core/server/master/src/test/java/alluxio/master/file/FileSystemMasterSyncMetadataMetricsTest.java

```diff
@@ -0,0 +1,705 @@
+    // The sync should succeed and the path /dir0/file0 should be removed from inodeTree
+    mUfs.mThrowIOException = true;
+    syncScheme = new LockingScheme(new AlluxioURI(path), InodeTree.LockPattern.READ,
+        true); // shouldSync
+    syncStream = new InodeSyncStream(syncScheme, mFileSystemMaster, RpcContext.NOOP,
+        DescendantType.NONE, options,
+        false, //isGetFileInfo
+        false, // forceSync
+        false, // loadOnly
+        false); // loadAlways
+    syncStream.sync();
+
+    assertEquals(streamCount + 1, streamCountCounter.getCount());
+    streamCount += 1;
+    assertEquals(succeededStreams + 1, succeededStreamCounter.getCount());
+    succeededStreams += 1;
+    assertEquals(succeededPaths + 1, succeededPathCounter.getCount());
+    succeededPaths += 1;
+    assertEquals(failedStreams + 0, failedStreamCounter.getCount());
+    failedStreams += 0;
+    assertEquals(failedPaths + 0, failedPathCounter.getCount());
+    failedPaths += 0;
+    assertEquals(noChangedPaths + 0, noChangedPathsCounter.getCount());
+    noChangedPaths += 0;
+    assertEquals(skippedStreams + 0, skippedStreamCounter.getCount());
+    skippedStreams += 0;
+
+    // Now the path(/dir0/file0) is non-existent in inodeTree and existent in UFS
+    // The sync should fail because UFS throws IOException
+    mUfs.mThrowIOException = true;
+    syncScheme = new LockingScheme(new AlluxioURI(path), InodeTree.LockPattern.READ,
+        true); //shouldSync
+    syncStream = new InodeSyncStream(syncScheme, mFileSystemMaster, RpcContext.NOOP,
+        DescendantType.NONE, options,
+        false, //isGetFileInfo
+        false, //forceSync
+        false, //loadOnly
+        false); //loadAlways
+    syncStream.sync();
+
+    assertEquals(streamCount + 1, streamCountCounter.getCount());
+    streamCount += 1;
+    assertEquals(succeededStreams + 0, succeededStreamCounter.getCount());
+    succeededStreams += 0;
+    assertEquals(succeededPaths + 0, succeededPathCounter.getCount());
+    succeededPaths += 0;
+    assertEquals(failedStreams + 1, failedStreamCounter.getCount());
+    failedStreams += 1;
+    assertEquals(failedPaths + 1, failedPathCounter.getCount());
+    failedPaths += 1;
+    assertEquals(noChangedPaths + 0, noChangedPathsCounter.getCount());
+    noChangedPaths += 0;
+    assertEquals(skippedStreams + 0, skippedStreamCounter.getCount());
+    skippedStreams += 0;
+
+    // The path(/dir/file) are non-existent in inodeTree and UFS
+    // The sync should fail
+    mUfs.mThrowIOException = false;
+    syncScheme = new LockingScheme(new AlluxioURI(TEST_DIR_PREFIX + dirNum + TEST_FILE_PREFIX),
+        InodeTree.LockPattern.READ,
+        true); //shouldSync
+    syncStream = new InodeSyncStream(syncScheme, mFileSystemMaster, RpcContext.NOOP,
+        DescendantType.NONE, options,
+        false, //isGetFileInfo
+        false, //forceSync
+        false, // loadOnly
+        false); // loadAlways
```

Comment:
Check all other comments and make sure the space is there
```suggestion
        false, // isGetFileInfo
        false, // forceSync
        false, // loadOnly
        false); // loadAlways
```